frontend/src/components/AdminRulesPage.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Component } from 'react';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+import LogbarAuth from './LogbarAuth';
+import DoctorsListWithSomeRule from './DoctorsListWithSomeRule';
+
+class AdminRulesPage extends Component {
+    render () {
+        return (
+            <div>
+                <div>
+                    <LogbarAuth/>
+                </div>
+                <div>
+                    <RulesList/>
+                </div>
+            </div>
+        );
+    }
+}
+
+var server_url;
+/*if(process.env.NODE_ENV==="development")
```

Comment:
@olegRakus did you forget to uncomment ?

Reply:
@ira-halenok yes, sorry